integration_tests/src/main/python/udf_cudf_test.py

```diff
@@ -123,95 +132,98 @@ def gpu_run(spark):
     _assert_cpu_gpu(cpu_run, gpu_run, gpu_conf=_conf)
 
 
-@pytest.mark.skip("https://github.com/NVIDIA/spark-rapids/issues/746")
+# ======= Test Flat Map In Pandas =======
 @allow_non_gpu('GpuMapInPandasExec','PythonUDF')
 @cudf_udf
-def test_map_in_pandas():
+def test_map_in_pandas(enable_cudf_udf):
     def cpu_run(spark):
-        df = _create_df(spark)
         def _filter_cpu_func(iterator):
             for pdf in iterator:
                 yield pdf[pdf.id == 1]
+        df = _create_df(spark)
         return df.mapInPandas(_filter_cpu_func, df.schema).collect()
 
     def gpu_run(spark):
-        df = _create_df(spark)
         def _filter_gpu_func(iterator):
             import cudf
             for pdf in iterator:
                 gdf = cudf.from_pandas(pdf)
                 yield gdf[gdf.id == 1].to_pandas()
+        df = _create_df(spark)
         return df.mapInPandas(_filter_gpu_func, df.schema).collect()
     
     _assert_cpu_gpu(cpu_run, gpu_run, gpu_conf=_conf)
 
 
+# ======= Test Grouped Map In Pandas =======
 # To solve: Invalid udf: the udf argument must be a pandas_udf of type GROUPED_MAP
 # need to add udf type
 @pandas_udf("id long, v double", PandasUDFType.GROUPED_MAP)
 def _normalize_cpu_func(df):
     v = df.v
     return df.assign(v=(v - v.mean()) / v.std())
 
+
```

Comment:
remove extra newline

Reply:
Actually two lines between two functions is recommended by the IntelliJ IDE.